src/main/scala/quality.scala

```diff
@@ -102,19 +101,15 @@ case object Quality {
 
   final
   def scoreFrom(errP: ErrorP): Score =
-    // ((-10) * errP.log(10)).round().intValue
-    Math.round((-10:Double) * errP.log(10)).toInt
-
+    Math.round( -10 * Math.log10(errP) ).toInt
 
   final
   def errorProbability(n: Score): ErrorP =
     phred33ErrorPCache.getOrElse(n, errorProbabilityImpl(n))
 
   private final
   def errorProbabilityImpl(n: Score): ErrorP =
-    // BigDecimal(10) fpow ( - (BigDecimal(n) / 10) )
-    (10:Double) fpow ( - ((n:Double) / 10) )
-
+    Math.pow(10, -(n: Double) / 10)
```

Comment:
same as above. Use `10D` if you prefer that syntax.

Reply:
Done in 83da952